### PR TITLE
V1.3.0 update ug

### PR DESCRIPTION
**Pull request by YuunoKun**
Update UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -303,7 +303,23 @@ bye
 
 ### 5.3. Students
 
-[short overview]
+{ start of `Features#Students` written by: Zhang Sheng Yang }
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps `Features#Students` could be renamed to `features#students`? Same comment for line 322.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -478,7 +494,19 @@ Examples:
 
 ### 5.5. Current session
 
-[short overview]
+{ start of `Current session#overview` written by: Zhang Sheng Yang }
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps this could be renamed to `features#current_session`?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -568,15 +596,43 @@ Command | Format, Examples
 
 ### 7.2. Students
 
-[students]
+{ start of `Command summary#Students` written by: Zhang Sheng Yang }
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps this could be renamed to `command_summary#students`?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -478,7 +494,19 @@ Examples:
 
 ### 5.5. Current session
 
-[short overview]
+{ start of `Current session#overview` written by: Zhang Sheng Yang }
+<br/>
+<br/>
+After entering a session, which is mentioned above, you can now edit the presence and participation details
+of a student. `participate` allows you to toggle the participation status (from absent to present or from 
+present to absent), and `presence` allows you to toggle the presence status of a student(from not
```
Comment (erisjacey) [inline docs/UserGuide.md]: Maybe the opening "(" at the end of this line could have a space between the word "student" preceding it?

**Review by erisjacey — COMMENTED** [review]
LGTM! Just a few nitpicks.

**Review by nweiyue — APPROVED** [review]
LGTM!

**Pull request merged**